src/node/desktop/.eslintrc.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
     sourceType: 'module',
   },
 
-  plugins: ['@typescript-eslint', 'prettier'],
```

Comment:
I can back this out, but honestly, I find the 'prettier' warnings far too annoying / restrictive. The number of warnings spit out by VSCode when this is active is far too distracting and (compounded with the existing pain of Windows development) makes development on Windows just hugely unpleasant.

Reply:
No objections